server/models/client.js

```diff
@@ -287,13 +295,9 @@ var client = {
     request.send();
   },
 
-  setSettings: (payloads, callback) => {
+  setSettings (payloads, callback) {
     let request = new ClientRequest();
-
-    if (payloads.length === 0) {
-      callback({});
-      return;
-    }
+    if (payloads.length === 0) return callback({});
```

Comment:
is this meant to be `callback(null)` btw?

Reply:
No idea why I passed the empty object here. The UI doesn't do anything with this.